packages/dotcom-ui-data-embed/README.md

```diff
@@ -0,0 +1,88 @@
+# @financial-times/dotcom-ui-data-embed
+
+This package provides methods for putting data into your server-side rendered pages and safely retrieving it again in the browser.
```

Comment:
This is me mirroring language in `app-context` and `flags`, since the extra safety comes from the `DataEmbedStore` class rather than returning a regular object.